internal/utils/testutil/file.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutil
+
+import (
+	"os"
+	"strings"
+)
+
+func CreateTempFileCrt() string {
+	file := os.NewFile(0, "temp.crt")
+
+	newLocation := os.TempDir() + file.Name()
+
+	return strings.ReplaceAll(newLocation, `\`, `\\`)
+}
```

Comment:
This function seems not used. If it's not, please remove this file.

Reply:
adjusted